Started additional configuration tutorials

strongishllama opened the pull request:
- Added Cloud Provider Integration to the Documentation page's menu.
- Added Amazon Web Services (AWS) page under the Cloud Provider Integration menu item.
- Added Additional Configuration to the Tutorial page's menu.
- Added Emails, Logs and Uploads pages under the Additional Configuration menu item.
- Added Requirements page under the Installation menu item and moved the contents of the Tutorial's landing page.
- Updated the Tutorial's landing page as its content was meant to be in the Requirements page under Installation
- Updated Requirements pages to match Cmfive v3's requirements.
- Fixed various other title errors and ran formatter on files that were open to remove white space and check for spelling errors.

Commit:
Started additional configuration tutorials

Commit:
fix: Additional Configuration IAM Role update

Made updates to the emails, logging and upload tutorials to better outline the differences between running Cmfive in production vs development in relation to authentication with AWS.

Fixed several spelling mistakes in other miscellaneous files.

refs: 6980

adam-buckley reviewed (changes requested):
Wrong image in example, use Aussie spelling aye, fix **that** sentence

Commit:
fix: Pull request fixes

- Updated organization spelling to organisation
- Fixed several typos
- Updated file_transfer_tool image

refs: 6980

strongishllama commented:
> Wrong image in example, use Aussie spelling aye, fix **that** sentence

Requested changes implemented.
